src/main/java/model/Board.java

```diff
@@ -29,6 +43,21 @@ public void add(Entity entity) {
 		}
 	}
 
+	public void removeEntity(Entity entity) {
+		for (int i = entity.getMinorX(); i <= entity.getMayorX(); i++) {
+			for (int j = entity.getMinorY(); j <= entity.getMayorY(); j++) {
```

Comment:
Recordame cuando tengas bien resuelto el tema del movimiento, que te explique como hacer el refactor acá para no repetir estos 2 for siempre